src/controllers/trainee /Controller.ts

```diff
@@ -0,0 +1,85 @@
+import { Request , Response } from 'express';
+
+class TraineeController {
+static instance ;
+static getInstance = () => {
+if ( TraineeController.instance) {
+return TraineeController.instance;
+}
+TraineeController.instance = new TraineeController();
+return TraineeController.instance;
+}
+create = (req: Request , res: Response) => {
+    console.log('::::::::Create Trainee:::::::::::::::');
+    const Traineedata={
```

Comment:
do validation on the basis of required fields id and name along with proper status code, error message